src/NHSD.BuyingCatalogue.Identity.Api/Views/Consent/Index.cshtml

```diff
@@ -214,7 +209,7 @@
                                 <span data-test-id="agreement-error-message" asp-validation-for="AgreeWithTerms" class="nhsuk-error-message"></span>
                                 <div class="nhsuk-checkboxes">
                                     <div class="nhsuk-checkboxes__item">
-                                        <input data-test-id="agree-terms-checkbox" class="nhsuk-checkboxes__input" type="checkbox" asp-for="AgreeWithTerms" />
+                                        <input data-test-id="agree-terms-checkbox" class="nhsuk-checkboxes__input" type="checkbox" asp-for="AgreeWithTerms"/>
```

Comment:
I find self-closing tags are more readable with the space before the `/>` (as they were).

Reply:
👍